src/frontend/components/router-tree/router-tree.ts

```diff
@@ -52,7 +52,7 @@ export class RouterTree {
     };
 
     const nodes = this.tree(d3.hierarchy(
-      root.children.length > 1 ? root : root.children[0], d => d.children));
+      root.children.length === 0 || root.children.length > 1 ? root : root.children[0], d => d.children));
```

Comment:
I don't understand what you mean. You want brackets around the expression part of the ternary? Is it because it contains an OR? I don't see any issue of precedence (which I would bracket).

BTW, `root.children[0]` is an object (route config), not a boolean.